main.py

```
@@ -370,6 +389,44 @@ def _cleanup(done_task: asyncio.Task) -> None:
+                message, context=self.context, provider_id=provider_id
+            )
+            if not tags:
+                return
+
+            profile_dir = self.profile_injector.get_profile_dir(user_id)
+            os.makedirs(profile_dir, exist_ok=True)
+
+            for tag in tags:
+                fname = f"{tag['type']}.md"
+                filepath = os.path.join(profile_dir, fname)
+                # 内容相同则跳过写入，保持 mtime 有意义
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)